test_pull_requests

```diff
@@ -1006,11 +1103,37 @@ popd
             status = result == 'SUCCESS' ? 'success' : 'failure'
           end
         else
+          # in this case, we're in one of three states:
+          #  1) ACTION_NOT_MERGE: the pull request is not mergeable and needs a rebase
+          #  2) ACTION_NOT_TEAM: the pull request has no trusted triggers
+          #  3) the build has finished, with one of the following states:
+          #    - SUCCESS
+          #    - FAILURE
+          #    - ABORTED
+          #    - UNSTABLE
+          #    - NOTFOUND
+          #
+          # Regardless of the current state, since we have a trusted trigger, we want to
+          # re-submit this pull request for testing.
+          # TODO: technically, in states 1) and 2), we don't necessarily know if it's ever
```

Comment:
What's the harm in resubmit?  Here or the case you listed above.  Not sure it's worth the comments.